level/level.go

```diff
@@ -100,6 +129,36 @@ func allowed(allowed level) Option {
 	return func(l *logger) { l.allowed = allowed }
 }
 
+// Parse a string to it's corresponding level value. Valid strings are "debug",
+// "info", "warn", and "error". Strings are normalized via strings.TrimSpace and
+// strings.ToLower.
+func Parse(level string) (Value, error) {
+	switch strings.TrimSpace(strings.ToLower(level)) {
+	case debugValue.name:
+		return debugValue, nil
+
+	case infoValue.name:
+		return infoValue, nil
+
+	case warnValue.name:
+		return warnValue, nil
+
+	case errorValue.name:
+		return errorValue, nil
+
+	default:
+		return nil, ErrInvalidLevelString
+	}
+}
+
+// ParseDefault calls Parse and returns the default Value on error.
+func ParseDefault(level string, def Value) Value {
```

Comment:
```suggestion
func ParseDefault(level string, default Value) Value {
```

Spelling out the word default is more explicit and makes the docs match the declaration better.

Reply:
To be honest I'm not sure about use keywords as variable's names

Reply:
> To be honest I'm not sure about use keywords as variable's names

+1, I always try to avoid using any keyword or language token as a variable name.

Reply:
Ooops, I didn't think of that. Sorry for the confusion.